LoggingKata/TacoParser.cs

```diff
@@ -1,18 +1,32 @@
 ﻿namespace LoggingKata
 {
-    /// <summary>
-    /// Parses a POI file to locate all the Taco Bells
-    /// </summary>
     public class TacoParser
     {
         readonly ILog logger = new TacoLogger();
         
         public ITrackable Parse(string line)
         {
-            logger.LogInfo("Begin parsing");
 
-            // Do not fail if one record parsing fails, return null
-            return null; // TODO Implement
+            if (line == null)
+            {
+                return null;
+            }
+            TacoBell trackableTacoBell = new TacoBell();
+            Point Location = new Point();
+            var cells = line.Split(',');
+
+            if (cells.Length < 3 || line == null)
+            {
+                logger.LogInfo("LogError");
+                return null;
+            }
+
+            Location.Latitude = double.Parse(cells[0]);
+            Location.Longitude = double.Parse(cells[1]);
```

Comment:
These lines are risky- what if cells[0] or cells[1] can't be parsed as a double?  Consider using a TryParse here instead